app/lib/context-helper.js

```diff
@@ -1,11 +1,10 @@
-import { config } from '../config/index.js'
 import { PI_HUNT_AND_DAIRY_FOLLOW_UP_RELEASE_DATE } from '../constants/index.js'
 
-export const isPIHuntEnabledAndVisitDateAfterGoLive = (dateOfVisit) => {
+export const isVisitDateAfterGoLive = (dateOfVisit) => {
```

Comment:
I think this functions name still needs to mention PI hunt, as its hard to know what its talking about going live (you can see its on line 9, but the function probably ought to be named 'isVisitDateAfterPIHuntAndDairyGoLive' or something similar.